package.json

```diff
@@ -1,10 +1,10 @@
 {
   "name": "@carforyou/api-client",
-  "version": "9.1.0",
+  "version": "0.0.0",
   "description": "Api client to connect with our APIs",
   "scripts": {
     "version": "npm run build",
-    "build": "pika build",
+    "build": "rimraf pkg && rollup -c",
```

Comment:
can't you just use `rm` directly in an npm command? (I have in mind it's just invoking the shell)

Reply:
good question. rimraf has cross-OS support, whereas `rm` is a linux-specific command. I also remember it having some safety measures such as not deleting files outside of your root directory (ergo no "oops, I committed a `rm -rf /`" and now everyone is fucked) and dealing nicely with directories not existing in the first place